app/assets/javascripts/discourse/app/mixins/composer-upload.js

```diff
@@ -337,8 +337,14 @@ export default Mixin.create({
 
   @on("willDestroyElement")
   _unbindUploadTarget() {
+    if (this.uploadButton && this.uploadButtonEventListener) {
```

Comment:
You can do the same trick I have been showing you here:

this.uploadButton?.removeEvent...